packages/runtime-core/__tests__/helpers/renderSlot.spec.ts

```diff
@@ -84,5 +85,84 @@ describe('renderSlot', () => {
       expect(vnode.children).toEqual([fallback])
       expect(vnode.patchFlag).toBe(PatchFlags.BAIL)
     })
+
+    // #7713
```

Comment:
I don't really understand these tests. From what I can tell, the tests seem to be trying to emulate the output of the template compiler. I'm not sure that's the best way to test this change.

I think there are two approaches we could take for testing this:

1. We could just test `guardReactiveProps` in isolation.
2. We could have tests with an actual template and `v-bind` usage.

I think either would be fine.

Also, the proposed change doesn't just impact slots, so I think the tests should reflect that.